lm_eval/tasks/benchmarks/pythia.yaml

```diff
@@ -0,0 +1,12 @@
+group: pythia
```

Comment:
`pythia_paper`?

Reply:
Do you prefer `pythia_paper` to `pythia`? I thought `pythia` would be better 😂